ports/net/macathena-discuss/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup		mit-athena discuss 832df353d2b92d024aa78ddd6b8d0bfdea2a641c
```

Comment:
Please use a tag (and omit version and revision) if one exists instead of the commit hash.